lib/admin/app.rb

```diff
@@ -44,14 +44,25 @@ class App < Sinatra::Base
     post '/upload' do
       authenticate!
       begin
-        success, details = Services::ProcessFile.new(
-          params[:spreadsheet_file]
-        ).call
-
-        if success
-          slim :overview, locals: details
+        file = params[:spreadsheet_file]
+        process_file = Services::ProcessFile.new(file[:tempfile].path)
+
+        if process_file.call
+          slim :overview, locals: {
```

Comment:
there's a lot of locals being passed into the slim. Is this a smell that an object encapsulation is missing?